src/modules/tests/test-data/test-data.selector.ts

```diff
@@ -102,3 +106,32 @@ export const getSelectedTellMeQuestionText = (state: VehicleChecks) => {
 
 export const getShowMeQuestion = (state: VehicleChecks) =>
   showMeQuestions.find(question => question.code === get(state, 'showMeQuestion.code'));
+
+export const hasVehicleChecksBeenCompleted = (data: TestData): boolean => {
+  const showMeQuestionOutcome = data.vehicleChecks.showMeQuestion.outcome;
+  const tellMeQuestionOutcome = data.vehicleChecks.tellMeQuestion.outcome;
+
+  return (
+    showMeQuestionOutcome === CompetencyOutcome.P ||
+    showMeQuestionOutcome === CompetencyOutcome.DF ||
+    showMeQuestionOutcome === CompetencyOutcome.S ||
+    showMeQuestionOutcome === CompetencyOutcome.D)
+  && (
+      tellMeQuestionOutcome === CompetencyOutcome.P ||
+      tellMeQuestionOutcome === CompetencyOutcome.DF ||
+      tellMeQuestionOutcome === CompetencyOutcome.S ||
+      tellMeQuestionOutcome === CompetencyOutcome.D
+  );
+};
+
+export const getCatBLegalRequirements = (data: TestData): CatBLegalRequirements => {
+  return {
+    normalStop1: data.testRequirements.normalStart1 || false,
```

Comment:
confusing - normalstop1 and 2 are based on the result of normalStart1 and 2?
why the naming difference?
probably nothing wrong with this - may be my lack of familiarity with these competencies.

Reply:
Fixed - Nice Spot